test/cpp/end2end/flaky_network_test.cc

```diff
@@ -0,0 +1,413 @@
+/*
```

Comment:
please get a C person to review this file.

Reply:
Thanks, will do

Reply:
@muxi  can you please review the C++ test (this file)? The build stuff needs more work; I'm looking into that.

Reply:
changes LGTM but I also suggest asking for another review from core, e.g. yang or nico? :)